### ISSUE TITLE
[DOC] Description/usage tooltips

### ISSUE THREAD

**Issue by charlottegiseleweil** [EPFL-ENAC/resslab-astra_82001#14]
Can you add here all descriptions of variables, explanatory tooltips that should appear ? 
e.g. Alain @jondalar64 recommended to describe: 

- [ ] "Width d'après la SIA 261 c'est la surface accessible (donc la largeur de la dalle accessible au traffic)."
- [ ] Pour le alpha q2 (punching shear equal 0.40) -> phrase explicative de la section 7.2.2 ? "

> Pour l'étape 1, une valeur constante et actualisée de ɑQ1 et ɑQ2 est recommandée, et la
> valeur actualisée de ɑq est recherchée dans l'outil web. Des valeurs actualisées de 0.55 et
> 0.40 sont recommandées pour tous les contrôles, y compris le poinçonnement (*notez que
> l'on pourrait utiliser ɑQ2 = 0.35 pour les contrôles autres que le poinçonnement*), et les va-
> leurs actualisées de ɑq sont indiquées dans la Figure 6:72. Toutefois, une limite inférieure
> de 0.30 est recommandée pour ɑq . Il convient de noter que l'ingénieur pourra également
> choisir, suivant le cas, une valeur d'indice de fiabilité différente de β = 4.2."

- [ ] Explanatory text for Alpha Q as well ? 
- [ ] Explanatory text pour le Beta (reference appendix B) ? 
- [ ] Explanatory text for Phi (reference)?
- [ ] Traffic class reference to paragraph 4.3 and figure 4.10 of documentation  
- [ ] For the AE column, add the following explanation (taken from report) (The last 3 bullet points are not yet in the tool, awaiting for the data)
>Les efforts étudiés sont les suivants :
•	Moment de flexion M- au droit des appuis (poutre continue et dalle)
•	Moment de flexion M+ à mi-travée (poutre simple et dalle), le cas de la poutre conti-nue a été étudié mais n’est pas déterminant
•	Effort tranchant V aux extrémités de la travée centrale (poutre simple et dalle). 
•	Moment de flexion transversal MxMid à mi-travée (dalle)
•	Moment de flexion transversal MxEdg au droit des appuis (dalle)
•	Effort tranchant à l’encastrement/sur appuis (dalles de roulement)
•	Moment de flexion transversal à l’encastrement/sur appuis (dalles de roulement),
•	Moment transversal en travée (dalle entre poutres).

- [ ] alpha_Q2 for punching shear (local/Global)

**Comment by charlottegiseleweil** [comment]
@PaolaCostanzaMiglietta @Dimitrios-astra @Jondalar-64 - kind reminder that you can add explanatory tooltips that should appear on the tool  !

**Comment by PaolaCostanzaMiglietta** [comment]
View issue #36 last comment

**Comment by guilbep** [comment]
@PaolaCostanzaMiglietta Is it correct that we still don't have the content of the tooltip ? just their position ? If I'm wrong. I'm sorry I cannot find them. Could you point them to me, please ?

**Comment by PaolaCostanzaMiglietta** [comment]
@guilbep yes you are correct. The text for the info button is being finalized. I will send it over soon.

**Comment by PaolaCostanzaMiglietta** [comment]
@Dimitrios-astra It's the same I sent you on January 16. I think Pierre put it here as well just for convenience of references.

**Comment by guilbep** [comment]
@PaolaCostanzaMiglietta  @Dimitrios-astra  correct
I'm trying keep track of everything :)

**Comment by Dimitrios-astra** [comment]
To my point of you, it is ok, it corresponds to article 5.3.1.5 which explain the type of supports.

**Comment by PaolaCostanzaMiglietta** [comment]
Tool tips provided in #43 .
Not sure about the general text...

**Comment by charlottegiseleweil** [comment]
Ah thank you for the pointer, there was no assignee so we had missed this issue :oops: 
Thanks so much !! We'll be implementing this by Tuesday~ 
@guilbep

**Comment by PaolaCostanzaMiglietta** [comment]
I have taken a pass at this, in this Word document:

[About pop up text - PM.docx](https://github.com/user-attachments/files/18537246/About.pop.up.text.-.PM.docx)

Please @Dimitrios-astra take a look and modify as you see fit the English. I will update the Italian accordingly. French and German are out of my league (for now 😜)

@guilbep @charlottegiseleweil please add your names and credentials!

**Comment by charlottegiseleweil** [comment]
### (i) Locations : 

- i1 : next to traffic type title
- i2 : next to "Good quality road"
- i3 : next to REL
- i4 + i5 merged top-right box. "αQ2, local: i4 \n αQ2, global : i5"
- i6 see ppt
-  i7 : Just après beta = ...
- i8 : rajouter (i) à coté du séléctor
- i9 et i10 en haut à droite des blocs alpha. 
- i11 : next to title "Support conditions"
- i12 :  next to title "Longitudinal verification"


<img width="269" alt="Image" src="https://github.com/user-attachments/assets/53defffe-c73a-4176-a779-ca0b04265aad" />

<img width="455" alt="Image" src="https://github.com/user-attachments/assets/caaff049-0f7f-4b5e-831a-9ea1e76c3fae" />

**Comment by guilbep** [comment]
Thanks @charlottegiseleweil  it's done in  #9075eff  feat: enhance internationalization support and improve UI responsiveness with tooltips

<img width="1795" alt="Image" src="https://github.com/user-attachments/assets/29fb7774-c93e-40c6-a423-d13095417644" />

**Comment by Dimitrios-astra** [comment]
> I have taken a pass at this, in this Word document:
> 
> [About pop up text - PM.docx](https://github.com/user-attachments/files/18537246/About.pop.up.text.-.PM.docx)
> 
> Please [@Dimitrios-astra](https://github.com/Dimitrios-astra) take a look and modify as you see fit the English. I will update the Italian accordingly. French and German are out of my league (for now 😜)
> 
> [@guilbep](https://github.com/guilbep) [@charlottegiseleweil](https://github.com/charlottegiseleweil) please add your names and credentials!

Dear Paola, @PaolaCostanzaMiglietta could you send me the file directly, unfortunately I cannot acces the link for IT-security reasons. Thanks
